structs.go

```diff
@@ -650,7 +644,7 @@ type Role struct {
 	// The permissions of the role on the guild (doesn't include channel overrides).
 	// This is a combination of bit masks; the presence of a certain permission can
 	// be checked by performing a bitwise AND between this int and the permission.
-	Permissions int `json:"permissions"`
+	Permissions string `json:"permissions"`
```

Comment:
Perhaps instead of changing this type, we can use the `,string` property on the struct tag to have `encoding/json` automatically convert it for us?